idc/settings.py

```diff
@@ -514,7 +517,9 @@ def GET_BQ_COHORT_SETTINGS():
 # These settings allow use of MailGun as a simple API call
 EMAIL_SERVICE_API_URL = os.environ.get('EMAIL_SERVICE_API_URL', '')
 EMAIL_SERVICE_API_KEY = os.environ.get('EMAIL_SERVICE_API_KEY', '')
-NOTIFICATION_EMAIL_FROM_ADDRESS = os.environ.get('NOTIFICATOON_EMAIL_FROM_ADDRESS', 'info@canceridc.dev')
+#NOTIFICATION_EMAIL_FROM_ADDRESS = os.environ.get('NOTIFICATOON_EMAIL_FROM_ADDRESS', 'info@canceridc.dev')
+
+NOTIFICATION_EMAIL_FROM_ADDRESS = os.environ.get('NOTIFICATOON_EMAIL_FROM_ADDRESS', 'info@localhost')
 
```

Comment:
Same as above. -- we can't use dev-type values for defaults. You'll want to edit your .env to contain this value, not edit the default.